validator_worker/src/main.rs

```diff
@@ -195,15 +196,16 @@ async fn validator_tick<A: Adapter + 'static>(
     channel: Channel,
     config: &Config,
     logger: &Logger,
-) -> Result<(), ValidatorWorkerError<A::AdapterError>> {
+) -> Result<(ChannelId, Box<dyn Debug>), ValidatorWorkerError<A::AdapterError>> {
```

Comment:
why is it validator_tick returning `(ChannelId, Box<dyn Debug>)` if we don't need / use it?

Reply:
It's logged